### PR TITLE
Transaltion to Polish language

### PR DESCRIPTION
**Pull request by dMbski**
Transaltion to Polish language. 
Thanks for this addon.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+bin-install\
```
Comment (lieser) [inline .gitignore:1]: Not against creating a gitignore (probably about time, have one uncommitted locally).

But this folder is currently part of any "official" workflow. If you delete the readme, maybe ignoring `*.xpi` files would be enough?

**Review by lieser — CHANGES_REQUESTED** [review]
Thanks a lot for providing this new translation. Please let me know if you want to be credited by something else than your GitHub user name.

I have a small comment about he `.gitignore`, and you need to uncommit the files in `bin-install/`.

**Comment by dMbski** [comment]
Folder bin-install was for temporary use for addon install (from file). It'not need anymore.

File: .gitignore (hunk 1 above)
Comment (dMbski) [reply .gitignore]: deleting files and folder

**Review by lieser — APPROVED** [review]

**Pull request merged**